### PR TITLE
[WIP] i18n(fr): Add French tutorial

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

Tutorial translation to French.

### CODE REVIEW

File: src/content/docs/fr/tutorial/0-introduction/index.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Lede from '~/components/tutorial/Lede.astro';
+
+
+<Lede> Dans de tutoriel, vous apprendrez les fonctionnalites cles d'Astro en realisant un blog totalement fonctionnel, de zero jusqu'au deploiement en ligne! 🚀 </Lede>
+
+Sur votre chemin, vous:
```
Comment (sdp971) [inline src/content/docs/fr/tutorial/0-introduction/index.mdx:18]: I suggest : 

Au cours de ce tutoriel, vous serez amené(e) à :

- Mettre en place un environnement de développement
- Créer des pages et des articles de blog pour votre site web
- Utiliser des composants Astro pour construire votre site
- Interroger et manipuler des fichiers en local
- Ajouter de l'interactivité à votre site
- Mettre en ligne votre site sur Internet

File: src/content/docs/fr/tutorial/0-introduction/index.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Sur votre chemin, vous: 
+- Configurerez un environnement de developement 
+- Creerez des pages et des articles de blog posts pour votre site web
+- Construirez avec des composants Astro
+- Chercherez et travaillerez avec des fichiers locaux
+- Ajouterez de l'interactivite sur votre site
+- Deploierez votre site sur internet
+
+Voulez vous avoir un percu de ce que vous allez realiser? Vous pouvez voir le projet fini sur [GitHub](https://github.com/withastro/blog-tutorial-demo) ou [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
```
Comment (sdp971) [inline src/content/docs/fr/tutorial/0-introduction/index.mdx:26]: I suggest for line 26 => Souhaitez-vous avoir un aperçu de ce que vous allez réaliser ? Vous pouvez visualiser le projet finalisé à l'adresse suivante : [GitHub](https://github.com/withastro/blog-tutorial-demo) ou [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).

File: src/content/docs/fr/tutorial/0-introduction/index.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+:::note
+Si vous preferez commencer a explorer Astro avec une version pre-construite, vous pouvez visiter https://astro.new et selectionner un gabarit de demmarrage, l'ouvrir et l'editer dans l'editeur en ligne.
```
Comment (sdp971) [inline src/content/docs/fr/tutorial/0-introduction/index.mdx:29]: I suggest for line 29 => Si vous souhaitez plutôt commencer à explorer Astro avec un site Astro pré-construit, vous pouvez visiter https://astro.new et choisir un modèle, l'ouvrir et le modifier dans un éditeur en ligne.

File: src/content/docs/fr/tutorial/0-introduction/index.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+::: 
+
+
+<Box icon="check-list">
+## Checklist
+
+<Checklist>
+- [ ] Ca a l'air bon! Je suis pret a commencer!
```
Comment (sdp971) [inline src/content/docs/fr/tutorial/0-introduction/index.mdx:37]: I suggest for line 37 => Parfait, je suis prêt(e) à commencer !

**Review by sdp971 — CHANGES_REQUESTED** [review]
Hi, Good job ✌️. I made suggestions and rephrased few sentences. Hope it will help.

File: src/content/docs/fr/tutorial/0-introduction/index.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  dont vous avez besoin pour commencer!
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Lede from '~/components/tutorial/Lede.astro';
+
+
+<Lede> Dans de tutoriel, vous apprendrez les fonctionnalites cles d'Astro en realisant un blog totalement fonctionnel, de zero jusqu'au deploiement en ligne! 🚀 </Lede>
```
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/tutorial/0-introduction/index.mdx:16]: > Dans ce tutoriel, vous apprendrez les fonctionnalités clés d'Astro en réalisant un blog totalement fonctionnel à partir de zéro jusqu'à son déploiement en ligne.

This one is cool! ☕

**Review by thomasbnt — APPROVED** [review]

**Comment by thomasbnt** [comment]
Arrrg I approved, but this is a mistake, only comment and to be notified of any changes. 😶

**Comment by Yan-Thomas** [comment]
Thanks for the reviews @sdp971 & @thomasbnt, this is a "collection" PR that merges other folks translations, I particularly don't spreak French so I can't really discuss or know if what you proposed is appropriate, could you make a separate PR making these specific changes to this branch and then I can merge them here?

**Comment by sdp971** [comment]
> Thanks for the reviews @sdp971 & @thomasbnt, this is a "collection" PR that merges other folks translations, I particularly don't spreak French so I can't really discuss or know if what you proposed is appropriate, could you make a separate PR making these specific changes to this branch and then I can merge them here?

Hi @Yan-Thomas, ok I'll make the PR tonight. I might need some guidance during the process.

**Comment by thomasbnt** [comment]
> Hi @Yan-Thomas, ok I'll make the PR tonight. I might need some guidance during the process.

Hello! Thanks @sdp971 !

**Comment by sdp971** [comment]
> > Hi @Yan-Thomas, ok I'll make the PR tonight. I might need some guidance during the process.
> 
> Hello! Thanks @sdp971 !

You're welcome 🙂

**Comment by sdp971** [comment]
> Thanks for the reviews @sdp971 & @thomasbnt, this is a "collection" PR that merges other folks translations, I particularly don't spreak French so I can't really discuss or know if what you proposed is appropriate, could you make a separate PR making these specific changes to this branch and then I can merge them here?

@Yan-Thomas it's done  [here](https://github.com/withastro/docs/pull/4252). Hope it's fine.

**Comment by McFlyPartages** [comment]
What is the status of this PR ?

**Comment by Yan-Thomas** [comment]
> What is the status of this PR ?

This PR is just to allow people to see a deploy preview of the translation as it is translated in other, individual, PRs tackling certain pages of it. Some people have been translating the pages with time, so it is being actively maintained.

**Comment by Ranily57** [comment]
I sure did mistake haha, thank's for the help on the translation !

**Comment by thomasbnt** [comment]
Hello! Seems an old MR since https://github.com/withastro/docs/pull/4688 was created for tutorial part.

**Comment by sarah11918** [comment]
Yes, it appears that the other PR is the complete one! We will review it for merging this week! :rocket:

**Comment by sarah11918** [comment]
Just closing this PR in favour of the full French tutorial being merged in the other PR! :partying_face:

**Pull request closed without merging**